pyproject.toml

```diff
@@ -84,6 +84,7 @@ operatorcert-dev = [
     "types-requests>=2.31.0.2",
     "pylint>=2.17.5",
     "openshift-client>=1.0.19",
+    # "pymarkdownlnt>=0.9.13.4",
```

Comment:
Is this intentional change?

Reply:
I left that there intentionally, but it can be removed if you prefer cleaner pyproject.toml file. Shall I do that?